.github/native-tests.json

```diff
@@ -63,7 +63,7 @@
         {
             "category": "Messaging2",
             "timeout": 70,
```

Comment:
I think we should increase this value.

Reply:
That test suite finishes in ~15min, I think 70min timeout is still ok.